### PR TITLE
Feature/body services

### PR DESCRIPTION
**Pull request by DoVanTuan1204**

### CODE REVIEW

File: src/styles/layout/_banner.scss [hunk 1]
```diff
@@ -8,20 +8,16 @@
...
   overflow: hidden;
   .banner-img {
     position: absolute;
-
     z-index: 100;
     display: flex;
-    height: 504px;
-    width: 1440px;
+    width: 100%;
     left: 0px;
```
Comment (BaoTranid) [inline src/styles/layout/_banner.scss]: 0px => 0

File: src/styles/layout/_body.scss [hunk 2]
```diff
@@ -1,130 +1,174 @@
...
-    border-radius: 15px;
-    position: absolute;
-    margin-left: 300px;
-  }
-  .body-content {
-    width: 500px;
-    height: 360px;
-    background-color: #6823cf;
-    border-radius: 12px;
-    box-shadow: 0px 2px 24px rgba(0, 0, 0, 0.08);
```
Comment (BaoTranid) [inline src/styles/layout/_body.scss:31]: 0px => 0

File: src/styles/layout/_body.scss [hunk 3]
```diff
@@ -1,130 +1,174 @@
...
+      transform: translate(75%, 135%);
+    }
+    .numerricActive {
+      line-height: 24px;
+      font-weight: 700;
+      color: #2b2b2b;
+      cursor: pointer;
+    }
+    .numerricActive::after {
+      content: " ";
```
Comment (BaoTranid) [inline src/styles/layout/_body.scss]: " " => ""

File: src/styles/layout/_body.scss [hunk 4]
```diff
@@ -1,130 +1,174 @@
...
+    .body-griddots {
+      display: none;
+    }
+    .body-number {
+      display: none;
+    }
+    // body services
+    .body-services {
+      position: relative;
+      margin-top: 0%;
```
Comment (BaoTranid) [inline src/styles/layout/_body.scss]: 0

**Review by BaoTranid — CHANGES_REQUESTED** [review]

**Review by kienbx1 — APPROVED** [review]

**Review by BaoTranid — APPROVED** [review]

**Pull request merged**